### ISSUE TITLE
Reimplement !lastreplay

### ISSUE THREAD

**Issue by FemtoZetta** [FAForever/Dostya#20]
We had this command before, would be nice to add it again because it's very useful.

**Comment by chrsin** [comment]
@FemtoZetta Could I have you specify here: https://github.com/FAForever/faf-qai/issues/11
what the command should do? (I'm assuming it finds a link to the last replay of yourself (or a player you specify)) but if there is anything else that it should do it would really be helpful.

**Comment by FemtoZetta** [comment]
Yes, just the last replay of the player you give, for example like this !lastreplay FemtoZetta and then it shows the replay the same way as if I had done #replayid or !replay replayid
